features/user_can_login.feature

```diff
@@ -0,0 +1,31 @@
+Feature: User can log in
+
+As a user
+In order to access my account
+I would like to be able log in
+
+Background:
+        Given I am on the login page
```

Comment:
I need you to click your way to the login page, so we know that the link works

Reply:
ok